client/packages.go

```diff
@@ -91,10 +91,23 @@ type FindOptions struct {
 
 var ErrNoSnapsInstalled = errors.New("no snaps installed")
 
+type ListOptions struct {
+	All bool
+}
+
 // List returns the list of all snaps installed on the system
 // with names in the given list; if the list is empty, all snaps.
-func (client *Client) List(names []string) ([]*Snap, error) {
-	snaps, _, err := client.snapsFromPath("/v2/snaps", nil)
+func (client *Client) List(names []string, opts *ListOptions) ([]*Snap, error) {
+	if opts == nil {
+		opts = &ListOptions{}
+	}
+
+	q := make(url.Values)
+	if opts.All {
+		q.Add("all", "1")
```

Comment:
Indeed, `select={all,enabled}` feel nice (and more consistent with find). I updated the code but I'm keen to hear the opinion of @chipaca as well.